src/evict/evict_inline.h

```diff
@@ -614,17 +616,18 @@ __evict_check_user_ok_with_eviction(WT_SESSION_IMPL *session, bool busy)
  *     should assist eviction worker threads with eviction of pages from the queues.
  *
  *     Input parameters:
- *       (1) `busy`: A flag indicating if eviction is mandatory (true) or optional (false).
- *       (2) `readonly`: A flag indicating if the session is read-only, in which case dirty and
- *            update triggers are ignored.
- *       (3) `didworkp`: A pointer to indicate whether eviction work was done (optional).
+ *       - `busy`: A flag indicating if eviction is mandatory (true) or optional (false).
+ *       - `readonly`: A flag indicating if the session is read-only, in which case dirty and
+ *          update triggers are ignored.
+ *       - `interruptible`: A flag indicating if eviction is ok to be interrupted by user.
+ *       - `didworkp`: A pointer to indicate whether eviction work was done (optional).
```

Comment:
For consistency reasons could we leave this as (1), (2), ... and so on?

Reply:
I don’t think we have a style standard for this, and I’d prefer not to use numbers. They don’t add much value but do add maintenance overhead. They also force descriptions for all arguments - including those that don’t need them (like session). And they aren’t even correct anyway since session is skipped.

I’d rather switch to bullet points everywhere.

That said, this isn’t the focus of this PR, so if you feel strongly about keeping numbers, I’m okay with using them here.

Reply:
Consistency is valuable, I suggest changing it back to numbers and creating a ticket to use bullet points instead (I agree bullet points are better) of numbers as changing comment format is not the focus of this PR.

Reply:
Done